Neos.Flow/Classes/Http/RequestHandler.php

```diff
@@ -133,7 +125,7 @@ public function getHttpRequest()
      */
     public function getHttpResponse()
     {
-        return $this->componentContext->getHttpResponse();
+        return $this->httpResponse;
```

Comment:
This one is problematic, because the response doesn't even exist before the innermost middleware returns. So we'll deprecate this. Use a middleware if you want to interact with the HTTP Response directly.